internal/rootcoord/timeticksync.go

```diff
@@ -320,9 +319,6 @@ func (t *timetickSync) startWatch(wg *sync.WaitGroup) {
 
 // SendTimeTickToChannel send each channel's min timetick to msg stream
 func (t *timetickSync) sendTimeTickToChannel(chanNames []string, ts typeutil.Timestamp) error {
-	if streamingutil.IsStreamingServiceEnabled() {
```

Comment:
redundant code here, the timetickSync do not start watcher if streaming service is enabled.